interpolate_phase.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+
+"""
+Copyright (c) 2023-, California Institute of Technology ("Caltech"). U.S.
+
+Government sponsorship acknowledged.
+All rights reserved.
+Author (s): Emre Havazli
+"""
+
+import argparse
+import glob
+import os
+import shutil
+import time
+
+import matplotlib.pyplot as plt
+import numpy as np
+from osgeo import gdal
+
+EXAMPLE = '''
+interpolate_phase.py <path_to_ifg_folder> <coherence_threshold>
+<interpolation_method>
+'''
+
+
+def create_parser():
+    """Create parser for command line arguments."""
+    parser = argparse.ArgumentParser(
+        description='Interpolate phase',
+        formatter_class=argparse.RawTextHelpFormatter,
+        epilog=EXAMPLE)
+
+    parser.add_argument(
+        'folder', type=str, help='Interferogram folder path')
+
+    parser.add_argument(
+        'coh_thresh', type=float, help='Coherence threshold to select '
+        'reliable pixels')
+
+    parser.add_argument(
+        'interp_method', type=str, help='Interpolation method: chen, chen_v2, numba')
+
+    return parser
+
+
+def get_intf(intf_file):
+    """Get interferogram and coherence from files."""
+    f = intf_file.split('/')
+    f[-1] = 'interp_filt_fine.int'
+    interp_file = os.path.join('/', *f)
+    shutil.copy(intf_file, interp_file)
+    shutil.copy(intf_file + '.xml', interp_file + '.xml')
+    ds = gdal.Open(interp_file, gdal.GA_ReadOnly)
```

Comment:
Also maybe we could add a load gdal function and reduce the loading lines, maybe something like this?
`def gdal_load(path, band):
    """
    Load image file into array
    using gdal.
    
    Inputs
    ------
    path = path to image file, text.
    band = band to load, int.
    
    Returns
    -------
    loaded file as numpy array
    """
    ds = gdal.Open(path, gdal.GA_ReadOnly)
    arr = ds.GetRasterBand(band).ReadAsArray()
    ds = None
    # summary of load
    print('Loaded data, length =', arr.shape[0], ', width =', arr.shape[1], ', and data type =', arr.dtype)
    
    return arr`